sdk/storage/CONTRIBUTING.md

```diff
@@ -18,28 +18,35 @@ The Azure Storage development team uses Visual Studio Code. However, any preferr
+  | ACCOUNT_KEY               | Can be obtained from the Azure portal. Go to the `Access Keys` tab under the `Settings` section of your storage account in the portal and note down the key.                                                                                                                                                                                                                                                                                              | Required                                                                         |
+  | ACCOUNT_SAS               | Can be obtained from the Azure portal. Go to the `Shared access signature` tab under the `Settings` section of your storage account in the portal. Generate SAS by enabling permissions, setting exiry dates, etc. You can also generate a valid account SAS from tools like Azure Storage Explorer. A SAS starts with "?". And if you are using Windows CMD, you may need quotes to escape special characters like this - `set "ACCOUNT_SAS=<YOUR_SAS>"` | Required to run the browser tests                                                |
+  | ACCOUNT_TOKEN             | Can you provide some description refferring the proper way to get this, @jiacfan?                                                                                                                                                                                                                                                                                                                                                                         | Not required (Some of the tests will be skipped if this variable is not defined) |
+  | STORAGE_CONNECTION_STRING | Can be obtained from the Azure portal. Go to the `Access Keys` tab under the `Settings` section of your storage account in the portal and note down the connection string.                                                                                                                                                                                                                                                                                | Required                                                                         |
+  |                           |                                                                                                                                                                                                                                                                                                                                                                                                                                                           |                                                                                  |
```

Comment:
Can you provide some description referring the proper way to get ACCOUNT_TOKEN, @jiacfan?

Reply:
**[Synced-Up offline with @jiacfan]** 

<pre>
Preferred way is to get a Token through Azure CLI.
- az login
- az account get-access-token --resource "https://storage.azure.com"

For more info - refer [Azure CLI - get-access-token](https://docs.microsoft.com/en-us/cli/azure/account?view=azure-cli-latest#az-account-get-access-token)
</pre>
![image](https://user-images.githubusercontent.com/10452642/72561306-c8312780-385d-11ea-8274-69a12e846139.png)